src/mds/CDir.cc

```diff
@@ -1481,57 +1481,6 @@ void CDir::fetch(MDSInternalContextBase *c, const std::set<dentry_key_t>& keys)
   _omap_fetch(c, keys);
 }
 
-class C_IO_Dir_TMAP_Fetched : public CDirIOContext {
- public:
-  bufferlist bl;
-
-  C_IO_Dir_TMAP_Fetched(CDir *d) : CDirIOContext(d) { }
-  void finish(int r) {
-    dir->_tmap_fetched(bl, r);
-  }
-};
-
-void CDir::_tmap_fetch()
```

Comment:
The declarations of _tmap_fetch and _tmap_fetched need to be removed from CDir.h as well